mathesar_ui/src/components/cell/utils.ts

```diff
@@ -34,18 +34,19 @@ function getCellConfiguration(
   };
 }
 
-export function getCellComponentWithProps(
+export function getCellCap(
```

Comment:
Why the shortened name here? Is "CAP" Supposed to be an acronym for "Component And Props"? I would find this naming confusing, especially because "cap" is also a word with its own meaning. I much prefer the more verbose name.

Reply:
> Is "CAP" Supposed to be an acronym for "Component And Props"

Yes, it is. Keeping it verbose made it long, and I thought we could establish the term `CAP` in our code base as an acronym for `Component And Props`, since we don't use "cap" anywhere else. But I'm fine with keeping it verbose as well.